scripts/retrieve_databundle_light.py

```diff
@@ -121,63 +130,203 @@ def download_and_unzip(host, config, rootpath, hot_run=True):
     True when download is successful, False otherwise
 
     """
-    resource = "-".join(config["category"])
+    resource = config["category"]
     file_path = os.path.join(rootpath, "tempfile.zip")
 
-    if host == "zenodo":
-        url = config["urls"]["zenodo"]
-        if hot_run:
+    url = config["urls"]["zenodo"]
+    if hot_run:
+        try:
+            logger.info(
+                f"Downloading resource '{resource}' from cloud '{url}'")
             progress_retrieve(url, file_path)
             logger.info(f"Extracting resources")
             with ZipFile(file_path, "r") as zipObj:
                 # Extract all the contents of zip file in current directory
                 zipObj.extractall(path=config["destination"])
             os.remove(file_path)
-            logger.info(f"Download resource '{resource}' from cloud '{url}'.")
-        return True
-    elif host == "google":
-
-        url = config["urls"]["google"]
-        # retrieve file_id from path
-        # cut the part before the ending \view
-        partition_view = re.split(r"/view|\\view", str(url), 1)
-        if len(partition_view) < 2:
-            logger.error(
-                f'Resource {resource} cannot be downloaded: "\\view" not found in url {url}'
-            )
+            logger.info(
+                f"Downloaded resource '{resource}' from cloud '{url}'.")
+        except:
+            logger.warning(
+                f"Failed download resource '{resource}' from cloud '{url}'.")
             return False
 
-        # split url to get the file_id
-        code_split = re.split(r"\\|/", partition_view[0])
+    return True
+
+
+def download_and_unzip_protectedplanet(config, rootpath, hot_run=True):
+    """
+        download_and_unzip_protectedplanet(config, rootpath, dest_path, hot_run=True)
+
+    Function to download and unzip the data by category from protectedplanet
+
+    Inputs
+    ------
+    config : Dict
+        Configuration data for the category to download
+    rootpath : str
+        Absolute path of the repository
+    hot_run : Bool (default True)
+        When true the data are downloaded
+        When false, the workflow is run without downloading and unzipping
+
+    Outputs
+    -------
+    True when download is successful, False otherwise
+
+    """
+    resource = config["category"]
+    file_path = os.path.join(rootpath, "tempfile_wpda.zip")
+
+    url = config["urls"]["protectedplanet"]
```

Comment:
The config refers to the config_bundle.yaml ?
Where to find the protectedplanet URL?